modules/builds-image-source.adoc

```diff
@@ -55,6 +55,15 @@ source:
 <5> The location of the file to be copied out of the referenced image.
 <6> An optional secret provided if credentials are needed to access the input image.
 
+If an input image requires a pull secret, you may optionally link the pull secret to the service
```

Comment:
Suggested rewrite: 

Optionally, if an input image requires a pull secret you can link the pull secret to the service account used by the build.